client/src/router.js

```diff
@@ -0,0 +1,16 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import MapContainer from './components/MapContainer.vue'
+
+export default createRouter({
```

Comment:
Can we add a comment here just like /** Defines all routes for app. **/

Reply:
Good note thanks